planex/depend.py

```diff
@@ -167,7 +181,9 @@ def main():
                             defines=macros)
             pins[os.path.basename(pin_path)] = spec
 
-    for spec_path in args.specs:
+    links = {pkgname(lnk): lnk for lnk in args.specs if lnk.endswith(".lnk")}
+
+    for spec_path in [spec for spec in args.specs if spec.endswith(".spec")]:
```

Comment:
I've tweaked the help text, but I think I'll be changing the interface a bit more in a future pull request.  I think it might be best to pick up the sources from their expected locations, as rpmbuild does, rather than passing them on the command line.  Also, it is redundant to pass a link file and a spec file at the same time - if a link file is passed, we can probably assume that we can fetch the spec file from its usual location or maybe just get it from the patchqueue tarball.